flow/layers/layer_tree.cc

```diff
@@ -52,7 +53,8 @@ void LayerTree::UpdateScene(SceneUpdateContext& context,
 void LayerTree::Paint(CompositorContext::ScopedFrame& frame) {
   Layer::PaintContext context = {frame.canvas(), frame.context().frame_time(),
                                  frame.context().engine_time(),
-                                 frame.context().memory_usage()};
+                                 frame.context().memory_usage(),
+                                 checkerboard_offscreen_layers_};
```

Comment:
I don't notice similar updates to `scene_update_context.cc`. Are you sure you checked the Fuchsia build?